demo/app/controllers/application_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
+
+  def check_optimizely_client
```

Comment:
Why not rename it to 'optimizely_client_present?'

Reply:
Updated